hack/netpol/test-kind.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR=$THIS_DIR/../..
+
+KIND_CONFIG=$ROOT_DIR/ci/kind/config-3nodes.yml
+
+WAIT_TIMEOUT=15m
+JOB_NAME=job.batch/netpol
+
+echo "===> Creating Kind cluster <==="
+
+kind create cluster --config $KIND_CONFIG
+kind get nodes | xargs $ROOT_DIR/hack/kind-fix-networking.sh
+kind load docker-image antrea/antrea-ubuntu:latest
+kind load docker-image antrea/netpol:latest
+$ROOT_DIR/hack/generate-manifest.sh --kind | kubectl apply -f -
+
+echo "===> Creating netpol ServiceAccount and Job <==="
```

Comment:
do you want to be specific and also echo that a ClusterRoleBinding is being created too?

Reply:
Done, it is displayed by kubectl as well